vic/drivers/shared_all/src/initialize_veg.c

```diff
@@ -39,35 +39,44 @@ initialize_veg(veg_var_struct **veg_var,
 
     for (i = 0; i < Nveg; i++) {
         for (j = 0; j < options.SNOW_BAND; j++) {
+            // Prognostic states
             veg_var[i][j].albedo = 0.0;
             veg_var[i][j].displacement = 0.0;
             veg_var[i][j].fcanopy = 0.0;
             veg_var[i][j].LAI = 0.0;
             veg_var[i][j].roughness = 0.0;
-            veg_var[i][j].throughfall = 0.0;
             veg_var[i][j].Wdew = 0.0;
             veg_var[i][j].Wdmax = 0.0;
```

Comment:
Technically, Wdmax is a derived (or "diagnostic", I suppose) state, completely derived from LAI.  However, both LAI and Wdmax get updated at similar times (every time that LAI gets a new value, Wdmax is recomputed), so keeping the initialization of Wdmax here would be completely consistent with that other behavior.  It's your (and/or Bart and Joe's) call.